app/src/main/java/ch/epfl/sdp/drone3d/map/area/PolygonBuilder.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+/*
+ * This class was taken from the project Fly2Find and adapted for our project
+ */
+
+package ch.epfl.sdp.drone3d.map.area
+
+import com.mapbox.mapboxsdk.geometry.LatLng
+
+// TODO To be deleted if no strategy is implemented for it
```

Comment:
Is the TODO still needed ?

Reply:
Yes, I implemented a basic Polygon area builder because the maths to create Parallelogram area were not implemented. In the future we should remove this exemple and replace it with a parallelogram buidler.